unblob/processing.py

```diff
@@ -187,26 +198,27 @@ def _extract_chunk(
         outdir: Path,
         extractor: Extractor,
         result: TaskResult,
-    ):
-        # We only extract every blob once, it's a mistake to extract the same blog again
-        outdir.mkdir(parents=True, exist_ok=False)
+    ) -> bool:
+        # We only extract every blob once, it's a mistake to extract the same blob again
         try:
-            extractor.extract(inpath, outdir, result)
+            outdir.mkdir(parents=True, exist_ok=False)
+            extracted = extractor.extract(inpath, outdir, result)
+
+            fix_extracted_directory(outdir, result)
+
+            result.add_new_task(
+                Task(
+                    root=self._extract_root,
+                    path=outdir,
+                    depth=task.depth + 1,
+                )
+            )
```

Comment:
if extraction failed, shell we cleanup extracted files and process them? probably yes, just wondering about this

Reply:
Agree, and it was reverted